stock_batch_picking/wizard/batch_picking_creator.py

```diff
@@ -43,14 +43,9 @@ def _default_picker_id(self):
         else:
             warehouse = self.env['stock.warehouse'].search([
                 ('company_id', '=', self.env.user.company_id.id)
-            ], limit=1)
+            ], limit=1)[:1]
```

Comment:
@lasley  Right! And I changed it (again...)

As it was a very very little change, can we merge this PR ?